deployment/docker-compose/docker-compose.yml

```diff
@@ -59,6 +59,8 @@ services:
     volumes:
       - ./litellm_proxy_config.yaml:/app/config.yaml
       - ./.gcp_credentials.json:/app/credentials.json
+    ports: # Expose the port to port 4000 for debugging purposes
+      - 4000:4000
```

Comment:
Is this temporary?

Reply:
I made it for two reasons:
- To make debugging easier
- For the script. 
But if there is an option to easily debug I can reverse it after sending the script.